media/css/custom.scss

```diff
@@ -521,3 +526,32 @@ summary {
     color: $white;
   }
 }
+
+.dictionary-item a {
+  color: $white;
+  float: left;
+  font-size: 50px;
+  line-height: 80px;
+  list-style: none;
+  margin-left: 37px;
+  margin-top: 25px;
+  text-align: center;
+  text-decoration: none;
+  width: 80px;
+}
+
+.dictionary-active {
+  background: $sidebar-background;
+
+  &:hover {
+    background: $sidebar-link-highlight;
+  }
+}
+
+.dictionary-inactive {
+  background: $dictionary-inactive;
+}
+
+.dictionary-no-margin a:nth-child(8n+1) {
```

Comment:
What is `.dictionary-no-margin` class for? Wouldn't it be better if you just moved `a:nth-child` property definition under `.dictionary-container`?